examples/pipeline_wavernn/main.py

```diff
@@ -0,0 +1,403 @@
+import argparse
+import logging
+import os
+import signal
+from collections import defaultdict
+from datetime import datetime
+from time import time
+from typing import List
+
+import torch
+import torchaudio
+from torch import nn as nn
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+from torchaudio.datasets.utils import bg_iterator
+from torchaudio.models._wavernn import _WaveRNN
+
+from datasets import collate_factory, datasets_ljspeech
+from losses import MoLLoss
+from utils import MetricLogger, count_parameters, save_checkpoint
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```

Comment:
for later: if you train on another dataset, you could add a command line argument below to select the dataset to experiment with.